### PR TITLE
Moving FAQ - Datasets not downloading at all?

### PR DESCRIPTION
**Pull request by Melkeb**
Moved "Datasets not downloading at all" [FAQ](https://galaxyproject.org/support/download-data/#datasets-not-downloading-at-all)

### CODE REVIEW

File: faqs/galaxy/datasets_not_downloading_at_all [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: Datasets not downloading at all?
+area: datasets
```
Comment (annefou) [inline faqs/galaxy/datasets_not_downloading_at_all:3]: I am not sure we need a question mark here.

File: faqs/galaxy/datasets_not_downloading_at_all [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+1.Check to see if pop-ups are blocked by your web browser. Where to check can vary by browser and extensions.
+2.Double check your API key, if used. Go to **User > Preferences > Manage API key.**
+3.Check the sharing/permission status of the Datasets. Go to **Dataset > pencil icon {% icon galaxy-pencil %} > Edit attributes > Permissions.** If you do not see a "Permissions" tab, then you are not the owner of the data.
+
+
+Notes:
+- If the data was shared with you by someone else from a Shared History, or was copied from a Published History, be aware that there are multiple levels of data sharing permissions.
+- All data are set to "not shared" by default.
```
Comment (annefou) [inline faqs/galaxy/datasets_not_downloading_at_all:18]: maybe we want to keep "not shared" in bold?

File: faqs/galaxy/datasets_not_downloading_at_all [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- Global sharing, new/existing history and data sharing preferences can be modified by you at any time under **User > Preferences** for data that you own.
```
Comment (annefou) [inline faqs/galaxy/datasets_not_downloading_at_all:19]: I am not sure it is correct e.g. this setting will only change the defaults for new histories e.g. created after these settings have been stored.

**Review by annefou — APPROVED** [review]
It looks good to me. Thanks.

**Comment by beatrizserrano** [comment]
@Melkeb could be that the file extension (.md) is missing?

**Comment by beatrizserrano** [comment]
The linting is failing here because @Nurzhamalyrys is not in the contributors list

**Comment by Nurzhamalyrys** [comment]
> 

@beatrizserrano Hi! I added myself as a contributor.

**Comment by beatrizserrano** [comment]
thank you very much @Melkeb and @Nurzhamalyrys!!

**Pull request merged**